docs/src/pages/components/data-grid/editing/ValidateRowModelControlGrid.js

```diff
@@ -65,7 +65,7 @@ export default function ValidateRowModelControlGrid() {
         rows={rows}
         columns={columns}
         editRowsModel={editRowsModel}
-        onEditCellChange={handleEditCellChange}
+        onEditCellPropsChange={handleCellEditPropsChange}
```

Comment:
The demo is no longer working, test https://deploy-preview-2143--material-ui-x.netlify.app/components/data-grid/editing/#client-side-validation vs. https://material-ui.com/components/data-grid/editing/#client-side-validation.

Shouldn't we use? 

```suggestion
        onEditRowsModelChange={handleCellEditPropsChange}
```

Reply:
Fixed. All demos now use `onEditRowsModelChange`.